src/components/dialog-full-screen/__snapshots__/dialog-full-screen.spec.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`DialogFullScreen dialogTitle has a Ref to the scrollable content and the ref to content exists and contains html 1`] = `undefined`;
```

Comment:
This snapshot should have content, not `undefined`.

Reply:
Replaced with expected innerHTML

Reply:
The unit tests are failing at the moment for this
Did the test generate this for you or did you add it manually?

Reply:
Updated again to JSON